tslint.json

```diff
@@ -1,20 +1,16 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-        "tslint:recommended",
-        "tslint-no-unused-expression-chai"
-    ],
-    "jsRules": {},
-    "rules": {
-      "prettier": true
-    },
-    "rulesDirectory": [
-      "tslint-plugin-prettier"
-    ],
-    "linterOptions": {
-      "exclude": [
-        "**/node_modules/**",
-        "**/*.js"
-      ]
-    }
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended", "tslint-no-unused-expression-chai"],
+  "jsRules": {},
+  "rules": {
+    "prettier": true,
+    "interface-name": false,
+    "object-literal-sort-keys": false,
+    "arrow-parens": false,
+    "trailing-comma": [true, { "functions": "never" }]
```

Comment:
I had to change this to get it to work properly with what prettier was doing.

Reply:
Yeah, I've seen this too. Fine by me.